cypress/pages/simulator/editorPage.js

```diff
@@ -0,0 +1,56 @@
+export class EditorPage {
```

Comment:
is there a root element to the editor we could use ?
To limit and fasten the search of elements maybe.